### ISSUE TITLE
Show / make explicit export location

### ISSUE THREAD

**Issue by dacap** [aseprite/aseprite#4483]
Some recurrent UI inconvenience when we export (File > Export > Export As / Export Sprite Sheet) is to show/select the location where the export will be done.

Some ideas to show the path or {elements}:

![image](https://github.com/aseprite/aseprite/assets/39654/1b82fdb1-1952-4522-bf96-244c0801a9bc)

**Comment by dacap** [comment]
@Liebranca saw that you’ve selected this one, contact me next week to see what we can do, some progress was already made in #4389, the mockup in this issue doesn’t solve the main problem that maybe we should attack and might be as simple as just showing the output folder in other field (we’re open to ideas here).

**Comment by Liebranca** [comment]
@dacap Yes, I was thinking about having another field too, since a logical division is already there. It's more optimal for both parts to be modified independent of each other, I feel. The question then is how should this division be presented. I'll take note of any ideas.

**Comment by dacap** [comment]
I think a possible solution to include the Relative/Absolute options is showing a menu when the "..." button is pressed:

![image](https://github.com/user-attachments/assets/7f8c9f70-6bdd-41df-babe-cdf91140233f)

Where "Choose File" does the current action of just pressing the "..." button. And "Relative Path" / "Absolute Path" changes how the path is displayed in the Entry field. The relative path option could show the folder that is used as reference: `Relative Path to "{}"`

**Comment by Liebranca** [comment]
Hi @dacap! I implemented the menu in f481a9c, now I'd like to make a separate widget class to better encapsulate this (filepath entry + '...' button with the menu), so that it can be reused for `ExportSpriteSheetWindow` without duplicating code.

**Comment by dacap** [comment]
Hi @Liebranca, that'd be nice! You can create a new widget. About your commit, I highly recommend you to create a draft PR (it's easier to test a PR using the [gh](https://cli.github.com/) CLI tool for me).
